### PR TITLE
remove ncpupdatedata

### PR DESCRIPTION
**Pull request by ethandcosta**
NCPUpdateData should be removed from the endaq device schema now that we decided that the update data exists outside of the EBML header

### CODE REVIEW

**Review by JulianCampanella — APPROVED** [review]
Not used in FW so good to go on that end
(I'd wait for David's approval too to merge though)

**Comment by ethandcosta** [comment]
> Not used in FW so good to go on that end (I'd wait for David's approval too to merge though)

**Comment by ethandcosta** [comment]
All good, David said that if you approved that it's good to merge.

**Pull request merged**
